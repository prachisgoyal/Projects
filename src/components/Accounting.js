import React from "react";
import { View, Text } from "react-native";

const Accounting = () => {
  return (
    <View>
      <Text>Accounting</Text>
    </View>
  );
};

export default Accounting;
