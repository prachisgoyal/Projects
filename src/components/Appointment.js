import React from "react";
import { View, Text } from "react-native";

const Appointment = () => {
  return (
    <View>
      <Text>Appointments</Text>
    </View>
  );
};

export default Appointment;
