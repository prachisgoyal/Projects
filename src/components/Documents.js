import React from "react";
import { View, Text } from "react-native";

const Documents = () => {
  return (
    <View>
      <Text>Patient Documents</Text>
    </View>
  );
};

export default Documents;
