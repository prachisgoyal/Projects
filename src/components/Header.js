import React from "react";
import { SafeAreaView, StyleSheet, Text, Image } from "react-native";
import colors from "../../assets/colors/colors";
import FontAwesome5 from "react-native-vector-icons/FontAwesome5";

const Header = () => {
  return (
    <SafeAreaView style={styles.container}>
      <FontAwesome5
        name="file-medical"
        size={32}
        color="rgba(500, 50, 100, 1)"
      />
      <Text style={styles.fontstyle}>Welcome Jena</Text>
    </SafeAreaView>
  );
};

export default Header;

const styles = StyleSheet.create({
  container: {
    paddingBottom: 20,
    paddingLeft: 10,
  },
  fontstyle: {
    paddingTop: 5,
    color: colors.white,
    fontWeight: "bold",
    fontSize: 20,
  },
});
