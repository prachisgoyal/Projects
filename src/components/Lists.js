import React, { useState } from "react";
import {
  View as SafeAreaView,
  Text,
  StyleSheet,
  View,
  TouchableOpacity,
} from "react-native";
import colors from "../../assets/colors/colors";

const Lists = () => {
  return (
    <SafeAreaView>
      <View style={styles.container}>
        <Text style={styles.profileHeading}>Medications</Text>
        <View style={styles.MedicationsContainer}>
          <Text style={styles.profileHeading2}>Drug</Text>
          <Text style={styles.profileHeading2}>Start Date</Text>
          <Text style={styles.profileHeading2}>End Date</Text>
          <Text style={styles.profileHeading2}>Referrer</Text>
        </View>

        <Text style={styles.profileHeading}>Medications Allergy List</Text>
        <View style={styles.MedicationsContainer}>
          <Text style={styles.profileHeading2}>Title</Text>
          <Text style={styles.profileHeading2}>Reported Date</Text>
          <Text style={styles.profileHeading2}>Referrer</Text>
        </View>
        <Text style={styles.profileHeading}>Issues List</Text>
        <View style={styles.MedicationsContainer}>
          <Text style={styles.profileHeading2}>Title</Text>
          <Text style={styles.profileHeading2}>Reported Date</Text>
          <Text style={styles.profileHeading2}>Referrer</Text>
        </View>
        <Text style={styles.profileHeading}>Amendment Lists</Text>
        <Text style={styles.profileHeading2}>No Results</Text>
        <Text style={styles.profileHeading}>Lab Results</Text>
        <Text style={styles.profileHeading2}>No Results</Text>
      </View>
    </SafeAreaView>
  );
};

const styles = StyleSheet.create({
  container: {
    alignItems: "stretch",
    flexDirection: "column",
    justifyContent: "space-evenly",
    paddingHorizontal: 25,
  },
  MedicationsContainer: {
    alignItems: "stretch",
    flexDirection: "row",
    justifyContent: "space-around",
  },
  textStyle: {
    fontSize: 20,
    fontWeight: "500",
    color: colors.fontGray,
    alignSelf: "flex-start",
    paddingHorizontal: 20,
  },
  profileText: {
    color: colors.fontGray,
    fontSize: 14,
    paddingVertical: 2,
    paddingHorizontal: 2,
  },
  profileHeading: {
    color: colors.fontGray,
    fontSize: 18,
    paddingVertical: 1,
    backgroundColor: colors.primaryColor,
    fontWeight: "500",
    paddingHorizontal: 2,
    marginTop: 30,
  },
  profileHeading2: {
    color: colors.fontGray,
    fontSize: 16,
    paddingVertical: 1,
    paddingHorizontal: 2,
    alignItems: "flex-start",
  },
});

export default Lists;
