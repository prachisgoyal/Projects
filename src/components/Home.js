import React from "react";
import {
  View as SafeAreaView,
  Text,
  StyleSheet,
  View,
  TouchableOpacity,
} from "react-native";
import colors from "../../assets/colors/colors";
import Profile from "./Profile";
import Accounting from "./Accounting";
import Reports from "./Reports";
import Lists from "./Lists";
import Appointment from "./Appointment";
import Documents from "./Documents";
import Header from "./Header";

const Home = ({ navigation }) => {
  return (
    <SafeAreaView>
      <View style={styles.container}>
        <View style={styles.column}>
          {values.map((value) => (
            <TouchableOpacity
              data-testid={value}
              key={value}
              style={styles.touchable}
              onPress={() => navigation.navigate(value)}
            >
              <Text style={styles.label}>{value}</Text>
            </TouchableOpacity>
          ))}
        </View>
      </View>
    </SafeAreaView>
  );
};

const values = [
  "Profile",
  "Lists",
  "Documents",
  "Appointment",
  "Accounting",
  "Reports",
];

const styles = StyleSheet.create({
  container: {
    flex: 1,
    flexDirection: "column",
    paddingTop: 20,
    alignItems: "center",
  },
  column: {
    paddingTop: 20,
  },
  touchable: {
    paddingHorizontal: 20,
    paddingVertical: 20,
    borderRadius: 8,
    alignItems: "center",
    justifyContent: "space-evenly",
    marginHorizontal: "12%",
    minWidth: "50%",
    backgroundColor: colors.primaryColor,
    marginVertical: 25,
    alignSelf: "center",
  },
  label: {
    fontSize: 20,
    fontWeight: "500",
    color: colors.white,
    alignSelf: "center",
  },
});

export default Home;
