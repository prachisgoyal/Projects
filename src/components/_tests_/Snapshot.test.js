import React from "react";
import renderer from "react-test-renderer";
import Home from "../Home";
import Profile from "../Profile";
import ProfileEdit from "../ProfileEdit";
import Accounting from "../Accounting";
import Appointment from "../Appointment";
import Reports from "../Reports";
import Lists from "../Lists";
import Documents from "../Documents";
import Header from "../Header";

test("renders Home screen correctly", () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});

test("renders Appointment screen correctly", () => {
  const tree = renderer.create(<Appointment />).toJSON();
  expect(tree).toMatchSnapshot();
});
test("renders Profile screen correctly", () => {
  const tree = renderer.create(<Profile />).toJSON();
  expect(tree).toMatchSnapshot();
});

test("renders ProfileEdit screen correctly", () => {
  const tree = renderer.create(<ProfileEdit />).toJSON();
  expect(tree).toMatchSnapshot();
});
test("renders Reports screen correctly", () => {
  const tree = renderer.create(<Reports />).toJSON();
  expect(tree).toMatchSnapshot();
});

test("renders Lists screen correctly", () => {
  const tree = renderer.create(<Lists />).toJSON();
  expect(tree).toMatchSnapshot();
});
test("renders Documents screen correctly", () => {
  const tree = renderer.create(<Documents />).toJSON();
  expect(tree).toMatchSnapshot();
});

test("renders Header screen correctly", () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
