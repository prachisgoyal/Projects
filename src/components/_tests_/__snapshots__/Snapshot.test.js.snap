// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Appointment screen correctly 1`] = `
<div
  className="css-view-1dbjc4n"
>
  <div
    className="css-text-901oao"
    dir="auto"
  >
    Appointments
  </div>
</div>
`;

exports[`renders Documents screen correctly 1`] = `
<div
  className="css-view-1dbjc4n"
>
  <div
    className="css-text-901oao"
    dir="auto"
  >
    Patient Documents
  </div>
</div>
`;

exports[`renders Header screen correctly 1`] = `
<div
  className="css-view-1dbjc4n r-paddingBottom-k8qxaj r-paddingLeft-1hfyk0a r-paddingRight-hxflta r-paddingTop-10xqauy"
>
  <div
    className="css-text-901oao"
    dir="auto"
    style={
      Object {
        "color": "rgba(255,50,100,1.00)",
        "fontFamily": "FontAwesome5_Solid",
        "fontSize": "32px",
        "fontStyle": "normal",
        "fontWeight": "normal",
      }
    }
  >
    ï‘·
  </div>
  <div
    className="css-text-901oao r-color-jwli3a r-fontSize-adyw6z r-fontWeight-vw2c0b r-paddingTop-glunga"
    dir="auto"
  >
    Welcome Jena
  </div>
</div>
`;

exports[`renders Home screen correctly 1`] = `
<div
  className="css-view-1dbjc4n"
>
  <div
    className="css-view-1dbjc4n r-alignItems-1awozwy r-flex-13awgt0 r-flexDirection-eqz5dr r-paddingTop-1knelpx"
  >
    <div
      className="css-view-1dbjc4n r-paddingTop-1knelpx"
    >
      <div
        className="css-view-1dbjc4n r-alignItems-1awozwy r-alignSelf-1kihuf0 r-backgroundColor-1rf9rz2 r-borderRadius-1xfd6ze r-cursor-1loqt21 r-justifyContent-1d7fvdj r-marginHorizontal-1ludzj4 r-marginVertical-zccstu r-minWidth-1dfn399 r-paddingHorizontal-d9fdf6 r-paddingVertical-tvv088 r-touchAction-1otgn73 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
        onClick={[Function]}
        onContextMenu={[Function]}
        onKeyDown={[Function]}
        style={
          Object {
            "WebkitTransitionDuration": "0s",
            "transitionDuration": "0s",
          }
        }
        tabIndex="0"
      >
        <div
          data-testid="Profile"
        >
          <div
            className="css-text-901oao r-alignSelf-1kihuf0 r-color-jwli3a r-fontSize-adyw6z r-fontWeight-majxgm"
            dir="auto"
          >
            Profile
          </div>
        </div>
      </div>
      <div
        className="css-view-1dbjc4n r-alignItems-1awozwy r-alignSelf-1kihuf0 r-backgroundColor-1rf9rz2 r-borderRadius-1xfd6ze r-cursor-1loqt21 r-justifyContent-1d7fvdj r-marginHorizontal-1ludzj4 r-marginVertical-zccstu r-minWidth-1dfn399 r-paddingHorizontal-d9fdf6 r-paddingVertical-tvv088 r-touchAction-1otgn73 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
        onClick={[Function]}
        onContextMenu={[Function]}
        onKeyDown={[Function]}
        style={
          Object {
            "WebkitTransitionDuration": "0s",
            "transitionDuration": "0s",
          }
        }
        tabIndex="0"
      >
        <div
          data-testid="Lists"
        >
          <div
            className="css-text-901oao r-alignSelf-1kihuf0 r-color-jwli3a r-fontSize-adyw6z r-fontWeight-majxgm"
            dir="auto"
          >
            Lists
          </div>
        </div>
      </div>
      <div
        className="css-view-1dbjc4n r-alignItems-1awozwy r-alignSelf-1kihuf0 r-backgroundColor-1rf9rz2 r-borderRadius-1xfd6ze r-cursor-1loqt21 r-justifyContent-1d7fvdj r-marginHorizontal-1ludzj4 r-marginVertical-zccstu r-minWidth-1dfn399 r-paddingHorizontal-d9fdf6 r-paddingVertical-tvv088 r-touchAction-1otgn73 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
        onClick={[Function]}
        onContextMenu={[Function]}
        onKeyDown={[Function]}
        style={
          Object {
            "WebkitTransitionDuration": "0s",
            "transitionDuration": "0s",
          }
        }
        tabIndex="0"
      >
        <div
          data-testid="Documents"
        >
          <div
            className="css-text-901oao r-alignSelf-1kihuf0 r-color-jwli3a r-fontSize-adyw6z r-fontWeight-majxgm"
            dir="auto"
          >
            Documents
          </div>
        </div>
      </div>
      <div
        className="css-view-1dbjc4n r-alignItems-1awozwy r-alignSelf-1kihuf0 r-backgroundColor-1rf9rz2 r-borderRadius-1xfd6ze r-cursor-1loqt21 r-justifyContent-1d7fvdj r-marginHorizontal-1ludzj4 r-marginVertical-zccstu r-minWidth-1dfn399 r-paddingHorizontal-d9fdf6 r-paddingVertical-tvv088 r-touchAction-1otgn73 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
        onClick={[Function]}
        onContextMenu={[Function]}
        onKeyDown={[Function]}
        style={
          Object {
            "WebkitTransitionDuration": "0s",
            "transitionDuration": "0s",
          }
        }
        tabIndex="0"
      >
        <div
          data-testid="Appointment"
        >
          <div
            className="css-text-901oao r-alignSelf-1kihuf0 r-color-jwli3a r-fontSize-adyw6z r-fontWeight-majxgm"
            dir="auto"
          >
            Appointment
          </div>
        </div>
      </div>
      <div
        className="css-view-1dbjc4n r-alignItems-1awozwy r-alignSelf-1kihuf0 r-backgroundColor-1rf9rz2 r-borderRadius-1xfd6ze r-cursor-1loqt21 r-justifyContent-1d7fvdj r-marginHorizontal-1ludzj4 r-marginVertical-zccstu r-minWidth-1dfn399 r-paddingHorizontal-d9fdf6 r-paddingVertical-tvv088 r-touchAction-1otgn73 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
        onClick={[Function]}
        onContextMenu={[Function]}
        onKeyDown={[Function]}
        style={
          Object {
            "WebkitTransitionDuration": "0s",
            "transitionDuration": "0s",
          }
        }
        tabIndex="0"
      >
        <div
          data-testid="Accounting"
        >
          <div
            className="css-text-901oao r-alignSelf-1kihuf0 r-color-jwli3a r-fontSize-adyw6z r-fontWeight-majxgm"
            dir="auto"
          >
            Accounting
          </div>
        </div>
      </div>
      <div
        className="css-view-1dbjc4n r-alignItems-1awozwy r-alignSelf-1kihuf0 r-backgroundColor-1rf9rz2 r-borderRadius-1xfd6ze r-cursor-1loqt21 r-justifyContent-1d7fvdj r-marginHorizontal-1ludzj4 r-marginVertical-zccstu r-minWidth-1dfn399 r-paddingHorizontal-d9fdf6 r-paddingVertical-tvv088 r-touchAction-1otgn73 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
        onClick={[Function]}
        onContextMenu={[Function]}
        onKeyDown={[Function]}
        style={
          Object {
            "WebkitTransitionDuration": "0s",
            "transitionDuration": "0s",
          }
        }
        tabIndex="0"
      >
        <div
          data-testid="Reports"
        >
          <div
            className="css-text-901oao r-alignSelf-1kihuf0 r-color-jwli3a r-fontSize-adyw6z r-fontWeight-majxgm"
            dir="auto"
          >
            Reports
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`renders Lists screen correctly 1`] = `
<div
  className="css-view-1dbjc4n"
>
  <div
    className="css-view-1dbjc4n r-alignItems-1oszu61 r-flexDirection-eqz5dr r-justifyContent-1d7fvdj r-paddingHorizontal-w1sl8e"
  >
    <div
      className="css-text-901oao r-backgroundColor-1rf9rz2 r-color-v258g r-fontSize-1i10wst r-fontWeight-majxgm r-marginTop-6ity3w r-paddingHorizontal-1p4rafz r-paddingVertical-1cy8qmm"
      dir="auto"
    >
      Medications
    </div>
    <div
      className="css-view-1dbjc4n r-alignItems-1oszu61 r-flexDirection-18u37iz r-justifyContent-a2tzq0"
    >
      <div
        className="css-text-901oao r-alignItems-1habvwh r-color-v258g r-fontSize-ubezar r-paddingHorizontal-1p4rafz r-paddingVertical-1cy8qmm"
        dir="auto"
      >
        Drug
      </div>
      <div
        className="css-text-901oao r-alignItems-1habvwh r-color-v258g r-fontSize-ubezar r-paddingHorizontal-1p4rafz r-paddingVertical-1cy8qmm"
        dir="auto"
      >
        Start Date
      </div>
      <div
        className="css-text-901oao r-alignItems-1habvwh r-color-v258g r-fontSize-ubezar r-paddingHorizontal-1p4rafz r-paddingVertical-1cy8qmm"
        dir="auto"
      >
        End Date
      </div>
      <div
        className="css-text-901oao r-alignItems-1habvwh r-color-v258g r-fontSize-ubezar r-paddingHorizontal-1p4rafz r-paddingVertical-1cy8qmm"
        dir="auto"
      >
        Referrer
      </div>
    </div>
    <div
      className="css-text-901oao r-backgroundColor-1rf9rz2 r-color-v258g r-fontSize-1i10wst r-fontWeight-majxgm r-marginTop-6ity3w r-paddingHorizontal-1p4rafz r-paddingVertical-1cy8qmm"
      dir="auto"
    >
      Medications Allergy List
    </div>
    <div
      className="css-view-1dbjc4n r-alignItems-1oszu61 r-flexDirection-18u37iz r-justifyContent-a2tzq0"
    >
      <div
        className="css-text-901oao r-alignItems-1habvwh r-color-v258g r-fontSize-ubezar r-paddingHorizontal-1p4rafz r-paddingVertical-1cy8qmm"
        dir="auto"
      >
        Title
      </div>
      <div
        className="css-text-901oao r-alignItems-1habvwh r-color-v258g r-fontSize-ubezar r-paddingHorizontal-1p4rafz r-paddingVertical-1cy8qmm"
        dir="auto"
      >
        Reported Date
      </div>
      <div
        className="css-text-901oao r-alignItems-1habvwh r-color-v258g r-fontSize-ubezar r-paddingHorizontal-1p4rafz r-paddingVertical-1cy8qmm"
        dir="auto"
      >
        Referrer
      </div>
    </div>
    <div
      className="css-text-901oao r-backgroundColor-1rf9rz2 r-color-v258g r-fontSize-1i10wst r-fontWeight-majxgm r-marginTop-6ity3w r-paddingHorizontal-1p4rafz r-paddingVertical-1cy8qmm"
      dir="auto"
    >
      Issues List
    </div>
    <div
      className="css-view-1dbjc4n r-alignItems-1oszu61 r-flexDirection-18u37iz r-justifyContent-a2tzq0"
    >
      <div
        className="css-text-901oao r-alignItems-1habvwh r-color-v258g r-fontSize-ubezar r-paddingHorizontal-1p4rafz r-paddingVertical-1cy8qmm"
        dir="auto"
      >
        Title
      </div>
      <div
        className="css-text-901oao r-alignItems-1habvwh r-color-v258g r-fontSize-ubezar r-paddingHorizontal-1p4rafz r-paddingVertical-1cy8qmm"
        dir="auto"
      >
        Reported Date
      </div>
      <div
        className="css-text-901oao r-alignItems-1habvwh r-color-v258g r-fontSize-ubezar r-paddingHorizontal-1p4rafz r-paddingVertical-1cy8qmm"
        dir="auto"
      >
        Referrer
      </div>
    </div>
    <div
      className="css-text-901oao r-backgroundColor-1rf9rz2 r-color-v258g r-fontSize-1i10wst r-fontWeight-majxgm r-marginTop-6ity3w r-paddingHorizontal-1p4rafz r-paddingVertical-1cy8qmm"
      dir="auto"
    >
      Amendment Lists
    </div>
    <div
      className="css-text-901oao r-alignItems-1habvwh r-color-v258g r-fontSize-ubezar r-paddingHorizontal-1p4rafz r-paddingVertical-1cy8qmm"
      dir="auto"
    >
      No Results
    </div>
    <div
      className="css-text-901oao r-backgroundColor-1rf9rz2 r-color-v258g r-fontSize-1i10wst r-fontWeight-majxgm r-marginTop-6ity3w r-paddingHorizontal-1p4rafz r-paddingVertical-1cy8qmm"
      dir="auto"
    >
      Lab Results
    </div>
    <div
      className="css-text-901oao r-alignItems-1habvwh r-color-v258g r-fontSize-ubezar r-paddingHorizontal-1p4rafz r-paddingVertical-1cy8qmm"
      dir="auto"
    >
      No Results
    </div>
  </div>
</div>
`;

exports[`renders Profile screen correctly 1`] = `
<div
  className="css-view-1dbjc4n"
>
  <div
    className="css-view-1dbjc4n r-alignItems-1oszu61 r-flexDirection-18u37iz r-justifyContent-1wtj0ep r-marginTop-6ity3w"
  >
    <div
      className="css-text-901oao r-alignSelf-k200y r-color-v258g r-fontSize-adyw6z r-fontWeight-majxgm r-paddingHorizontal-d9fdf6"
      dir="auto"
    >
      Jena
       
      Doe
    </div>
    <div
      className="css-view-1dbjc4n r-alignSelf-173mn98 r-backgroundColor-1rf9rz2 r-borderRadius-1xfd6ze r-cursor-1loqt21 r-marginRight-acz6fz r-maxWidth-xllap1 r-paddingHorizontal-1j3t67a r-touchAction-1otgn73 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
      onClick={[Function]}
      onContextMenu={[Function]}
      onKeyDown={[Function]}
      style={
        Object {
          "WebkitTransitionDuration": "0s",
          "transitionDuration": "0s",
        }
      }
      tabIndex="0"
    >
      <div
        className="css-text-901oao r-color-jwli3a r-fontSize-adyw6z r-fontWeight-majxgm"
        dir="auto"
      >
        Edit
      </div>
    </div>
  </div>
  <div
    className="css-view-1dbjc4n r-alignItems-1oszu61 r-flexDirection-eqz5dr r-justifyContent-1d7fvdj r-marginTop-1x0uki6 r-paddingHorizontal-w1sl8e"
  >
    <div
      className="css-text-901oao r-backgroundColor-1rf9rz2 r-color-v258g r-fontSize-1i10wst r-fontWeight-majxgm r-paddingHorizontal-1p4rafz r-paddingVertical-1cy8qmm"
      dir="auto"
    >
      Profile Demographics
    </div>
    <div
      className="css-text-901oao r-backgroundColor-1pnm7ga r-color-v258g r-fontSize-ubezar r-paddingHorizontal-1p4rafz r-paddingVertical-1cy8qmm"
      dir="auto"
    >
      Who
    </div>
    <div
      className="css-text-901oao r-color-v258g r-fontSize-1b43r93 r-paddingHorizontal-1p4rafz r-paddingVertical-1vvnge1"
      dir="auto"
    >
      female
    </div>
    <div
      className="css-text-901oao r-color-v258g r-fontSize-1b43r93 r-paddingHorizontal-1p4rafz r-paddingVertical-1vvnge1"
      dir="auto"
    >
      1985-02-09
    </div>
    <div
      className="css-text-901oao r-color-v258g r-fontSize-1b43r93 r-paddingHorizontal-1p4rafz r-paddingVertical-1vvnge1"
      dir="auto"
    >
      Single
    </div>
    <div
      className="css-text-901oao r-backgroundColor-1pnm7ga r-color-v258g r-fontSize-ubezar r-paddingHorizontal-1p4rafz r-paddingVertical-1cy8qmm"
      dir="auto"
    >
      Contact
    </div>
    <div
      className="css-text-901oao r-color-v258g r-fontSize-1b43r93 r-paddingHorizontal-1p4rafz r-paddingVertical-1vvnge1"
      dir="auto"
    >
      7 Rainbow
    </div>
    <div
      className="css-text-901oao r-color-v258g r-fontSize-1b43r93 r-paddingHorizontal-1p4rafz r-paddingVertical-1vvnge1"
      dir="auto"
    >
      Hamilton
    </div>
    <div
      className="css-text-901oao r-color-v258g r-fontSize-1b43r93 r-paddingHorizontal-1p4rafz r-paddingVertical-1vvnge1"
      dir="auto"
    >
      Ontario
    </div>
    <div
      className="css-text-901oao r-color-v258g r-fontSize-1b43r93 r-paddingHorizontal-1p4rafz r-paddingVertical-1vvnge1"
      dir="auto"
    >
      L20 KE2
    </div>
    <div
      className="css-text-901oao r-color-v258g r-fontSize-1b43r93 r-paddingHorizontal-1p4rafz r-paddingVertical-1vvnge1"
      dir="auto"
    >
      Canada
    </div>
    <div
      className="css-text-901oao r-color-v258g r-fontSize-1b43r93 r-paddingHorizontal-1p4rafz r-paddingVertical-1vvnge1"
      dir="auto"
    >
      patient@email.ca
    </div>
    <div
      className="css-text-901oao r-backgroundColor-1pnm7ga r-color-v258g r-fontSize-ubezar r-paddingHorizontal-1p4rafz r-paddingVertical-1cy8qmm"
      dir="auto"
    >
      Choices
    </div>
    <div
      className="css-text-901oao r-color-v258g r-fontSize-1b43r93 r-paddingHorizontal-1p4rafz r-paddingVertical-1vvnge1"
      dir="auto"
    >
      Provider
    </div>
  </div>
</div>
`;

exports[`renders ProfileEdit screen correctly 1`] = `
<div
  className="css-view-1dbjc4n"
>
  <div
    className="css-view-1dbjc4n r-alignItems-1oszu61 r-flexDirection-eqz5dr r-justifyContent-1d7fvdj r-marginTop-1x0uki6 r-paddingHorizontal-w1sl8e"
  >
    <div
      className="css-view-1dbjc4n r-alignSelf-173mn98 r-backgroundColor-1rf9rz2 r-borderRadius-1xfd6ze r-cursor-1loqt21 r-maxWidth-r2y082 r-paddingHorizontal-1j3t67a r-touchAction-1otgn73 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
      onClick={[Function]}
      onContextMenu={[Function]}
      onKeyDown={[Function]}
      style={
        Object {
          "WebkitTransitionDuration": "0s",
          "transitionDuration": "0s",
        }
      }
      tabIndex="0"
    >
      <div
        className="css-text-901oao r-color-jwli3a r-fontSize-adyw6z r-fontWeight-majxgm"
        dir="auto"
      >
        Save
      </div>
    </div>
    <input
      autoCapitalize="sentences"
      autoComplete="on"
      autoCorrect="on"
      className="css-textinput-11aywtz r-backgroundColor-1pnm7ga r-borderColor-1drizk4 r-marginVertical-zg41ew r-paddingHorizontal-utggzx r-paddingVertical-1vvnge1 r-placeholderTextColor-8rsm64"
      dir="auto"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onSelect={[Function]}
      placeholder="First Name"
      readOnly={false}
      spellCheck={true}
      type="text"
      value=""
    />
    <input
      autoCapitalize="sentences"
      autoComplete="on"
      autoCorrect="on"
      className="css-textinput-11aywtz r-backgroundColor-1pnm7ga r-borderColor-1drizk4 r-marginVertical-zg41ew r-paddingHorizontal-utggzx r-paddingVertical-1vvnge1 r-placeholderTextColor-8rsm64"
      dir="auto"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onSelect={[Function]}
      placeholder="last Name"
      readOnly={false}
      spellCheck={true}
      type="text"
      value=""
    />
    <input
      autoCapitalize="sentences"
      autoComplete="on"
      autoCorrect="on"
      className="css-textinput-11aywtz r-backgroundColor-1pnm7ga r-borderColor-1drizk4 r-marginVertical-zg41ew r-paddingHorizontal-utggzx r-paddingVertical-1vvnge1 r-placeholderTextColor-8rsm64"
      dir="auto"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onSelect={[Function]}
      placeholder="Gender"
      readOnly={false}
      spellCheck={true}
      type="text"
      value=""
    />
    <input
      autoCapitalize="sentences"
      autoComplete="on"
      autoCorrect="on"
      className="css-textinput-11aywtz r-backgroundColor-1pnm7ga r-borderColor-1drizk4 r-marginVertical-zg41ew r-paddingHorizontal-utggzx r-paddingVertical-1vvnge1 r-placeholderTextColor-8rsm64"
      dir="auto"
      inputMode="numeric"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onSelect={[Function]}
      placeholder="DOB: mm-dd-yyyy"
      readOnly={false}
      spellCheck={true}
      value=""
    />
    <input
      autoCapitalize="sentences"
      autoComplete="on"
      autoCorrect="on"
      className="css-textinput-11aywtz r-backgroundColor-1pnm7ga r-borderColor-1drizk4 r-marginVertical-zg41ew r-paddingHorizontal-utggzx r-paddingVertical-1vvnge1 r-placeholderTextColor-8rsm64"
      dir="auto"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onSelect={[Function]}
      placeholder="Marital status"
      readOnly={false}
      spellCheck={true}
      type="text"
      value=""
    />
    <input
      autoCapitalize="sentences"
      autoComplete="on"
      autoCorrect="on"
      className="css-textinput-11aywtz r-backgroundColor-1pnm7ga r-borderColor-1drizk4 r-marginVertical-zg41ew r-paddingHorizontal-utggzx r-paddingVertical-1vvnge1 r-placeholderTextColor-8rsm64"
      dir="auto"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onSelect={[Function]}
      placeholder="Address"
      readOnly={false}
      spellCheck={true}
      type="text"
      value=""
    />
    <input
      autoCapitalize="sentences"
      autoComplete="on"
      autoCorrect="on"
      className="css-textinput-11aywtz r-backgroundColor-1pnm7ga r-borderColor-1drizk4 r-marginVertical-zg41ew r-paddingHorizontal-utggzx r-paddingVertical-1vvnge1 r-placeholderTextColor-8rsm64"
      dir="auto"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onSelect={[Function]}
      placeholder="City"
      readOnly={false}
      spellCheck={true}
      type="text"
      value=""
    />
    <input
      autoCapitalize="sentences"
      autoComplete="on"
      autoCorrect="on"
      className="css-textinput-11aywtz r-backgroundColor-1pnm7ga r-borderColor-1drizk4 r-marginVertical-zg41ew r-paddingHorizontal-utggzx r-paddingVertical-1vvnge1 r-placeholderTextColor-8rsm64"
      dir="auto"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onSelect={[Function]}
      placeholder="State"
      readOnly={false}
      spellCheck={true}
      type="text"
      value=""
    />
    <input
      autoCapitalize="sentences"
      autoComplete="on"
      autoCorrect="on"
      className="css-textinput-11aywtz r-backgroundColor-1pnm7ga r-borderColor-1drizk4 r-marginVertical-zg41ew r-paddingHorizontal-utggzx r-paddingVertical-1vvnge1 r-placeholderTextColor-8rsm64"
      dir="auto"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onSelect={[Function]}
      placeholder="Postal Code"
      readOnly={false}
      spellCheck={true}
      type="text"
      value=""
    />
    <input
      autoCapitalize="sentences"
      autoComplete="on"
      autoCorrect="on"
      className="css-textinput-11aywtz r-backgroundColor-1pnm7ga r-borderColor-1drizk4 r-marginVertical-zg41ew r-paddingHorizontal-utggzx r-paddingVertical-1vvnge1 r-placeholderTextColor-8rsm64"
      dir="auto"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onSelect={[Function]}
      placeholder="Country"
      readOnly={false}
      spellCheck={true}
      type="text"
      value=""
    />
    <input
      autoCapitalize="sentences"
      autoComplete="on"
      autoCorrect="on"
      className="css-textinput-11aywtz r-backgroundColor-1pnm7ga r-borderColor-1drizk4 r-marginVertical-zg41ew r-paddingHorizontal-utggzx r-paddingVertical-1vvnge1 r-placeholderTextColor-8rsm64"
      dir="auto"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onSelect={[Function]}
      placeholder="Contact Email"
      readOnly={false}
      spellCheck={true}
      type="email"
      value=""
    />
  </div>
</div>
`;

exports[`renders Reports screen correctly 1`] = `
<div
  className="css-view-1dbjc4n"
>
  <div
    className="css-text-901oao"
    dir="auto"
  >
    Reports
  </div>
</div>
`;
